docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1238,919 @@ import 'handsontable/dist/handsontable.full.min.css';
-            return (value, nextValue) => 0; // Don't sort the first visual column.
-          }
-        }
-      }, {
-        data: 'model' // 2nd column is simple text, no special options here
-      }, {
-        data: 'maxSpeed',
-        type: 'numeric'
-      }, {
-        data: 'range',
-        type: 'numeric'
-      }, {
-        data: 'seats',
-        type: 'numeric'
-      }, {
-        data: 'price',
-        type: 'numeric',
-        numericFormat: {
-          pattern: '$ 0,0.00',
-          culture: 'en-US'
-        }
-      }, {
-        data: 'productionDate',
-        type: 'date',
-        dateFormat: 'MM/DD/YYYY',
-        correctFormat: true,
-        defaultDate: '01/01/1900'
-      }]}
-      columnSorting={true}
+      colHeaders={[
+        'Brand',
+        'Model',
+        'Price',
+        'Date',
+        'Time',
+        'In stock',
+      ]}
+      columnSorting={{
+        // at initialization, sort rows by column 1, in ascending order
+        initialConfig: {
+          column: 0,
+          sortOrder: 'asc',
+        },
+      }}
+      rowHeaders={true}
+      height="auto"
+      width="auto"
       licenseKey="non-commercial-and-evaluation"
     />
   );
 };
 
 /* start:skip-in-preview */
-ReactDOM.render(<ExampleComponent />, document.getElementById('example4'));
+ReactDOM.render(<YourHandsontableComponent />, document.getElementById('example4'));
 /* end:skip-in-preview */
 ```
 :::
-:::
 
+:::
 
-## Custom sort implementation
+## Add custom sort icons
 
-The plugin exposes the [`setSortConfig`](@/api/columnSorting.md#setsortconfig) method. Use this to set the internal sort state.
+To replace the default sort icons, overwrite the `.columnSorting.ascending::before`
+and `.columnSorting.descending::before` pseudo-elements of Handsontable's CSS.
 
-::: tip
-The data set **won't be sorted** just by using this method. The [`isSorted`](@/api/columnSorting.md#issorted) and [`getSortConfig`](@/api/columnSorting.md#getsortconfig) methods will return results corresponding to the previously set configuration.
-:::
+```css
+/* the icon for ascending order */
+.handsontable span.colHeader.columnSorting.ascending::before
 
-To use a custom sort implementation, you need to:
-* Provide a callback for the [`beforeColumnSort`](@/api/hooks.md#beforecolumnsort) hook, which will return `false`
-* Sort data by yourself - for example, on the server-side
-* Set the internal state of the sort by using the [`setSortConfig`](@/api/columnSorting.md#setsortconfig) method
+/* the icon for descending order */
+.handsontable span.colHeader.columnSorting.descending::before
+```
 
-The code snippet below provides an example of a custom sort implementation:
+This demo replaces the default sort icons with the 👆 and 👇 emojis.
 
 ::: only-for javascript
+
+::: example #example5 --css 1 --js 2
+
+```css
+/* the icon for ascending order */
+.your-handsontable-instance span.colHeader.columnSorting.ascending::before {
```

Comment:
`.your-handsontable-instance` doesn't make sense here. I can have a custom class, but that doesn't mean it should be included here.

Let's replace it with `.handsontable` class.

Also, there are two methods to replace that default icon. **The recommended way** should be to replace the default image (currently, it's a base64 image, but it could be plain PNG).

**The alternative way** should be to place there a custom emoji through the 'content` property in CSS.

In the alternative method, I would replace the 👆 icon with △ and 👇 with ▽.

Also, let's use the demo with refreshed data set in this section.

Reply:
Ok, done: [1201b6f](https://github.com/handsontable/handsontable/pull/10183/commits/1201b6fd6915c303dc49cb2cb9735a5ac21f8d5f)

Mind that when we change the `background-image` property, the icon displays too close to the column name

Reply:
> Mind that when we change the background-image property, the icon displays too close to the column name

@kirszenbaum If the new icon has a size other than the original `height: 10px`; `width: 5px`, you need to adjust CSS - [view the demo](https://jsfiddle.net/70jLmxnc/25/).

Some arrows in base64 for the testing purposes are available here: https://iconmonstr.com/angel-up-circle-thin-svg/ (click embed > base64)

Reply:
Right, thanks! Fixed: [398b56a](https://github.com/handsontable/handsontable/pull/10183/commits/398b56acc314ca830651ff5dd90da5a08122444c)